App/Helpers/Formatter.php

```diff
@@ -47,6 +48,26 @@ public static function dateIso2Fr($date)
         if (!preg_match('#^(\d{4})-(\d{2})-(\d{2})$#', $date, $matches)) {
             throw new \Exception('Date mal formée');
         }
+
         return $matches[3] . '/' . $matches[2] . '/' . $matches[1];
     }
+
+    /**
+     * Transforme une heure en un timestamp positionné sur le début du temps unix
+     *
+     * @param string $hour
+     *
+     * @return int
+     * @throws \Exception si le paramètre d'entrée n'est pas bien formé
+     * @since  1.9
+     */
+    public static function hour2Time($hour)
+    {
+        $pattern = '/^(((0?|1)[0-9])|(2[0-3])):([0-5][0-9])$/';
+        if (!preg_match($pattern, $hour, $matches)) {
+            throw new \Exception('Heure mal formée');
```

Comment:
![doh](https://sophosnews.files.wordpress.com/2009/01/the-simpsons-d-oh-mini-posters-71133.jpg)